DigitalOffice/Services/UserService/Commands/LoginUserCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UserService.Commands.Interfaces;
+using UserService.Database;
+using UserService.Database.Entities;
+using UserService.Models;
+using UserService.Validators;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using System.Security.Cryptography;
+using System.Text;
+using System.Net;
+using System.Web;
+using UserService.Controllers;
+
+namespace UserService.Commands
+{
+    public class LoginUserCommand : ILoginUserCommand
+    {
+        IValidator<UserCredentials> validator;
+        UserServiceDbContext dbContext;
+        User user;
```

Comment:
no need for class field, move inside method

Reply:
fixed